doc/dune-files.rst

```diff
@@ -685,18 +677,18 @@ to use the :ref:`include_subdirs` stanza.
 
 Note that when binding C libraries, dune doesn't provide special support for
```

Comment:
Dune capitalised here. :) 
Then in the next sentence: …such as ``pkg-config``; however, it integrated…